app/assets/stylesheets/modules/pagination.scss

```diff
@@ -1,11 +1,19 @@
 .pagination {
   text-align: center;
+  font-variant: small-caps;
+  font-size: 18px;
```

Comment:
We've been using `rem`s instead of `px`s, and we've been basing font sizes off of [`$base-font-size`](https://github.com/BaltimorePublicArtCommons/baltimore_public_art_commons/blob/master/app/assets/stylesheets/global_variables.scss#L1). Can we do something [like this](https://github.com/BaltimorePublicArtCommons/baltimore_public_art_commons/blob/master/app/assets/stylesheets/base.scss#L16) here?